NXP-33390: Align nuxeo-web-ui package version on npm dependencies

ataillefer opened the pull request.

ataillefer committed:
NXP-33390: Align nuxeo-web-ui package version on npm dependencies

kevinleturc commented on ci/docker/Dockerfile:
```diff
@@ -5,6 +5,7 @@ FROM docker-private.packages.nuxeo.com/nuxeo/nuxeo:${NUXEO_VERSION}
 ARG BUILD_TAG=unknown
 ARG SCM_REF=unknown
 ARG VERSION=unknown
+ARG NUXEO_WEB_UI_VERSION=unknown
```
This makes the image unbuildable if we don't give the version, I would be in favor of:

```suggestion
ARG NUXEO_WEB_UI_MP=nuxeo-web-ui
```

kevinleturc commented on ci/Jenkinsfiles/build.groovy:
```diff
@@ -202,6 +217,7 @@ pipeline {
             def clid = nxK8s.getSecretData(namespace: 'platform', name: clidSecret, key: 'instance\\.clid')
             def connectUrl = clidSecret.contains('preprod') ? CONNECT_PREPROD_SITE_URL : CONNECT_PROD_SITE_URL
 
+            echo "Web UI version read from npm dependencies: ${NUXEO_WEB_UI_VERSION}"
```
No need to log it imho, otherwise it opens the question about NUXEO_VERSION and others.

If we need to debug that we would be able to easily retrieve it from the docker image.
Also, as everything is fixed, like the pom, the version should be known in almost every cases.

kevinleturc replied:
Bis: if we want to log it, move the version retrieval just before the Docker image build stage and give it to nxDocker.build as envVars parameter.

As it we tackle two aspects:
- the version is logged
- npm should have been downloaded and installed at this moment (although I'm a bit surprise that it is working, didn't remember that npm was available in the builder image, I though we've removed it when switching to frontend maven plugin)

ataillefer replied:
Sure about logging. Indeed the npm call is't working, in fact I was thinking about switching to the playwright container.

ataillefer replied:
Or just `jq` directly on the `package-lock.json`, more straight-forward.

ataillefer committed:
NXP-33390: Downgrade `@nuxeo` dependencies to fix ftests

Waiting for https://hyland.atlassian.net/browse/WEBUI-1858 to be fixed.
Dependabot should update these dependencies to the latest version and manage to merge once the bug is fixed.

guirenard reviewed (approved):
Didn't we want to downgrade to nuxeo-web-ui `2025.6.0` (i.e., more precisely, downgrade `nuxeo-web-ui-ftest` and co to `2025.6.0`) to have a green build?
Dependabot will eventually bump it whenever https://hyland.atlassian.net/browse/WEBUI-1858 is fixed and released. Plus, we will be able to submit changes to this addon with a full CI check in the meantime.

**EDITED** ok you just did it while I was writing my comment

ataillefer commented:
> Didn't we want to downgrade to nuxeo-web-ui `2025.6.0` (i.e., more precisely, downgrade `nuxeo-web-ui-ftest` and co to `2025.6.0`) to have a green build? Dependabot will eventually bump it whenever https://hyland.atlassian.net/browse/WEBUI-1858 is fixed and released. Plus, we will be able to submit changes to this addon with a full CI check in the meantime.
> 
> **EDITED** ok you just did it while I was writing my comment

Yes exatcly, Kevin also pointed this out. In fact, Dependabot will keep trying with failing PRs, until the fix version is released.
